modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -0,0 +1,83 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	ibctesting "github.com/cosmos/ibc-go/testing"
+)
+
+func (suite *KeeperTestSuite) TestTrySendTx() {
+	var (
+		path   *ibctesting.Path
+		msg    interface{}
+		portID string
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success", func() {
```

Comment:
should there be a test for sending multiple messages?

Reply:
Updated :+1: